hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -208,24 +208,25 @@ genContestMutation
                           healthySignature (fromInteger mutatedSnapshotNumber)
                     }
               ]
-      , SomeMutation Nothing MutateRequiredSigner <$> do
+      , SomeMutation (Just "signer is not a participant") MutateRequiredSigner <$> do
           newSigner <- verificationKeyHash <$> genVerificationKey
           pure $ ChangeRequiredSigners [newSigner]
-      , SomeMutation Nothing MutateContestUTxOHash . ChangeOutput 0 <$> do
+      , SomeMutation (Just "too old snapshot") MutateContestUTxOHash . ChangeOutput 0 <$> do
```

Comment:
This should also fail with the signature not being consistent with the target utxo hash.

Reply:
Solved by [commit](https://github.com/input-output-hk/hydra/pull/719/commits/9ca9bed1232f11a835b7d1d60c80656af1fbe500).